rosidl_generator_cpp/msg/PrimitivesConstants.msg

```diff
@@ -0,0 +1,14 @@
+bool BOOL_CONST=true
+byte BYTE_CONST=50
+char CHAR_CONST=100
+float32 FLOAT32_CONST=1.125
+float64 FLOAT64_CONST=3.14159
```

Comment:
This is the value we use for `test_communication` because it has the exact same representation in both languages (c.f. https://github.com/ros2/system_tests/commit/0e2dad0d2f4570a7f097c1a58e33433eb815d8cc)
So I think this should be ok